extensions/notebook/src/test/model/notebookManager.test.ts

```diff
@@ -35,12 +35,13 @@ describe('Jupyter Notebook Manager', function (): void {
 	beforeEach(() => {
 		mockExtensionContext = new MockExtensionContext();
 		mockApiWrapper = TypeMoq.Mock.ofType(ApiWrapper);
+		mockApiWrapper.setup(w => w.setCommandContext(TypeMoq.It.isAnyString(), TypeMoq.It.isAny()));
```

Comment:
Why do you need to mock this out? You don't appear to be using the mock and these tests run within an ADS instance so it can access the actual context variables just fine.

Reply:
The constructor calls apiWrapper.setCommandContext, so anywhere we create a JupyterServerInstallation object it throws an error without this. ApiWrapper can probably just be a singleton to get around stuff like this.

Reply:
Although if we actually need to mock out the ApiWrapper methods, then singleton would be problematic.

Reply:
So you should be able to use ofInstance and then create a `new ApiWrapper()` - that will allow you to override only the calls you need to.

But also I think you could just remove the setContext from the ApiWrapper. I don't see any good reason to have that in the wrapper since there's never really a situation I think we need to actually mock that out (we can always manually set the context within the scope of a test if needed)